### ISSUE TITLE
merge github.com/lokkersp/terraform-provider-sops?

### ISSUE THREAD

**Issue by nwmcsween** [carlpett/terraform-provider-sops#91]
As per title this fork provides a sop_file resource which is handy to enc and dec directly using terraform.

**Comment by carlpett** [comment]
I agree that is a nice feature. However, it isn't using the public API of SOPS (since one does not exist), which I'm not that keen on doing.

**Comment by k11h-de** [comment]
I would love to see this feature being added.
Thanks for your effort and work here

**Comment by nick4fake** [comment]
@carlpett What about possibly adding it marking resources as temporary/experimental?

Right now we use null resources for encryption, what is extremely slow (like firstly downloading sops, for example)
